mmeds/database/database.py

```diff
@@ -1036,7 +1036,6 @@ def get_sequencing_run_locations(self, metadata, user, column=("RawDataProtocol"
         for run in df[column]:
             if run not in runs:
```

Comment:
this conditional confuses me

Reply:
you're right this is bad, it's an old bit of code from before I knew about the unique() function, fixed it now.